mesa/space.py

```diff
@@ -778,11 +778,119 @@ def out_of_bounds(self, pos):
 
 class NetworkGrid:
     """ Network Grid where each node contains zero or more agents. """
+    """ kwargs = the arguments to the nx generator function
+        e.g for watts_strogatz_graph(n, k, p[, seed])"""
+    def __init__(self, G=None, generator="Graph", **kwargs):
```

Comment:
The generator should be an object that defaults to `nx.Graph`. This is in line with `networkx`'s `read_edgelist`'s `create_using` argument https://networkx.github.io/documentation/networkx-1.9/reference/generated/networkx.readwrite.edgelist.read_edgelist.html.